pages/mensa.vue

```diff
@@ -53,23 +53,15 @@ export default {
       title: 'Mensa Plan',
     };
   },
-  // async fetch({ store }) {
-  //   store.dispatch('mensa/loadWeek');
-  // },
-  computed: {
-    ...mapState({
-      weekPlan: (state) => state.mensa.weekPlan,
-    })
-  },
-  mounted() {
-      this.loadWeek();
+  async fetch({ store, params }) {
+    await store.dispatch('mensa/loadWeek');
```

Comment:
Wenn das mit netlify generiert wird, bezieht sich die aktuelle Woche auf die build-Zeit. Im schedule.vue habe ich dafür das lazyLoad-Attribut eingeführt. Das könntest du aus dem `store/splus.js` nach `store/index.js` heben und wir verwenden das dann global.
Dann sollte für den lazy-Fall in mounted() nachgeladen werden.

Reply:
gut Idee